guides/plugins/plugins/testing/end-to-end-testing.md

```diff
@@ -8,11 +8,11 @@ In end-to-end testing \(E2E testing in short\) real user workflows are simulated
 
 To use Shopware E2E tests, at first you need to have a Shopware 6 installation running. Making sure, that your tests are reliable, you should have a clean installation. Cleanup means no categories, no products, no settings, nothing!
 
-The easiest way to clean up your installation is the initialization. Using the command `./psh.phar init` Shopware 6 gets initialized clean and without demo data. Installation of E2E dependencies can be accomplished separately by running `npm install` in the E2E folder you're using, e.g. for Shopware Administration it's `src/Administration/Resources/app/administration/test/e2e`.
+The easiest way to clean up your installation is the initialization. Using the command `composer run init` Shopware 6 gets initialized clean and without demo data. Installation of E2E dependencies can be accomplished separately by running `npm install` in the E2E folder you're using, e.g. for Shopware Administration it's `src/Administration/Resources/app/administration/test/e2e`.
```

Comment:
I'm not sure with all the docker mentioned, what to execute outside or inside and if it is even needed anymore cc @leichteckig can you please check that?